tests/console/procps.pm

```diff
@@ -48,6 +59,57 @@ sub run {
         qr/PID\sTTY\s+TIME\sCMD\s+1\s\?\s+\d+:\d+:\d+\s(systemd|init)/);
     validate_script_output("top -b -n 1",
 qr/top - \d+:\d+:\d+ up\s+((\d+:\d+)|(\d+ \w+)|(\d+ \w+,\s+\d+:\d+)),\s+\d+ \w+,\s+load average: \d+.\d+, \d+.\d+, \d+.\d+\s+Tasks:\s+\d+\s+total,\s+\d+\s+running,\s+\d+\s+sleeping.*top/s);
+
+    # Extend vmstat utility. Verifying the vmstat with options.
+    validate_script_output("vmstat -t",
+        qr/(procs\s-+memory-+\s-+swap-+\s-+io-+\s-+system-+\s-+cpu-+\s-+timestamp-+).*(\s+|\d+)+/s);
+    validate_script_output("vmstat -a",
+        qr/(procs\s-+memory-+\s-+swap-+\s-+io-+\s-+system-+\s-+cpu-+).*(\s+.*\s+inact\s+active\s*.*).*(\s+|\d+)+/s);
+    validate_script_output("vmstat -f", qr/(\d+\s+forks)/s);
+
+    #Running vmstat under light load and dumping the output to a file. Calculate and collect the minimum idle memory from the output.
+    assert_script_run("vmstat 2 10 > /tmp/output_light.log");
+    assert_script_run("cat /tmp/output_light.log");
+    my ($mem_lite_load_ref, $cpu_lite_load_ref) = read_memory_cpu("/tmp/output_light.log");
+    my @mem_lite_load     = @$mem_lite_load_ref;
+    my @cpu_lite_load     = @$cpu_lite_load_ref;
+    my $min_mem_lightload = min(@mem_lite_load);
+    my $min_cpu_lightload = min(@cpu_lite_load);
+
+    # Increasing the load in the system using package stress-ng.
+    add_suseconnect_product(get_addon_fullname('phub')) if (is_sle);
+    zypper_call('in stress-ng');
+    assert_script_run("rpm -q stress-ng");
+    my $total_cpu = script_output("cat /proc/cpuinfo | grep processor | wc -l");
+    my $total_mem = script_output("free -h | awk '/Mem\:/ { print \$2 }'");
+    if ($total_cpu == 1) { $cpu = $total_cpu; } else { $cpu = floor($cpu * 0.75); }
+    $vm = floor($total_mem * 0.75);
+    my $stressng_pkg = "nohup stress-ng --cpu $cpu --vm $vm --vm-bytes 512M --timeout 60s&";
```

Comment:
So this will run approx. 768 stress workers each using 512M ?
on my physical desktop with 16G RAM `nohup stress-ng --cpu 1 --vm 768 --vm-bytes 512M --timeout 60s&` caused 500 load, please try load VM not kill it.
What's the point of `/tmp/output_load.log` when you don't upload it ?
```
-m N, --vm N
              start N workers continuously calling mmap(2)/munmap(2) and writing to the allocated memory. Note that this can cause systems to trip the kernel OOM killer on Linux systems if not enough physical memory and swap is  not
              available.

       --vm-bytes N
              mmap N bytes per vm worker, the default is 256MB. One can specify the size as % of total available memory or in units of Bytes, KBytes, MBytes and GBytes using the suffix b, k, m or g.
```

Reply:
Ignore the part about `/tmp/output_load.log` it's just vmstat and it's ok as it is.

Reply:
I have modified stress workers , Now script will run with 50 workers with 512k per vm worker. Let me know your feedback.